From: Alec Leamas <leamas.alec@gmail.com>
Date: Sun, 21 Jul 2024 03:35:57 +0200
Subject: CMakeLists: handle help_web.html location3

Not acceptable upstream, this is a consequence of us removing the manual.

Forwarded: not-needed
---
 CMakeLists.txt | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 4aec2fe..bf697eb 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1840,7 +1840,7 @@ if (APPLE)
   )
 
   set_source_files_properties(
-    data/doc/help_web.html PROPERTIES MACOSX_PACKAGE_LOCATION SharedSupport/doc
+    data/help_web.html PROPERTIES MACOSX_PACKAGE_LOCATION SharedSupport/doc
   )
 
   if (OCPN_BUNDLE_DOCS)
@@ -1880,11 +1880,11 @@ if (APPLE)
   set(MACOSX_BUNDLE_GUI_IDENTIFIER "org.opencpn")
 
   if (OCPN_BUNDLE_DOCS)
-    set(DOC_FILES data/doc/help_web.html
+    set(DOC_FILES data/help_web.html
                   ${WIKI_DOC_FILES}
     )
   else ()
-    set(DOC_FILES data/doc/help_web.html)
+    set(DOC_FILES data/help_web.html)
   endif ()
 
   target_sources(
@@ -2049,7 +2049,7 @@ if (NOT APPLE)
   if (OCPN_BUNDLE_DOCS)
     install(DIRECTORY data/doc DESTINATION ${PREFIX_PKGDATA})
   else ()
-    install(FILES data/doc/help_web.html DESTINATION ${PREFIX_PKGDATA}/doc)
+    install(FILES data/help_web.html DESTINATION ${PREFIX_PKGDATA}/doc)
   endif ()
 
   if (UNIX)
